lib/rx/client.rb

```diff
@@ -44,14 +44,14 @@ def post_refill_rx(id)
       perform(:post, "prescription/rxrefill/#{id}", nil, token_headers)
     end
 
-    # TODO: Might need error handeling around this.
+    # TODO: Might need error handling around this.
     def get_preferences
       response = {}
       config.parallel_connection.in_parallel do
         response.merge!(get_notification_email_address)
         response.merge!(rx_flag: get_rx_preference_flag[:flag])
       end
-      { data: response, errors: {}, metadata: {} }
+      PrescriptionPreference.new(response)
```

Comment:
Actually, I think this will prove useful as we can layer on validations for "invalid email address" types on our end if we can lock down on the business logic MHV uses. I think they just look for no special charachters and va.gov domain.